src/Illuminate/Notifications/RoutesNotifications.php

```diff
@@ -20,6 +20,8 @@ public function notify($instance)
 
     /**
      * Get the notification routing information for the given driver.
+     * In case of an empty return value, the notification will
+     * not be sent.
```

Comment:
yes!.. it would be bad indeed, i'm not objecting to the way it's set up. not at all! 

I just think this little piece of comment would help some developers with debugging in the future.